scripts/kernels/debian.sh

```diff
@@ -1,32 +1,30 @@
 #! /bin/sh
 
-REPO="linuxkit/kernel-debian"
-BASE_URL=http://mirrors.kernel.org/debian/pool/main/l/linux/
-
-TAGS=$(curl --silent -f -lSL https://registry.hub.docker.com/v1/repositories/${REPO}/tags)
-
+if [ "$#" -ne 3 ]; then
+    echo "Usage: $0 <org/repo> <first version> <second version>"
```

Comment:
FWIW `<first version>` is really the ABI version while `<second version>` is the actual package version (assuming I've interpreted the script's behaviour correctly).